sgf-viewer/src/content.ts

```diff
@@ -179,6 +179,24 @@ export const pages: object =
                 "adversary": "498 million training steps, 600 visits",
                 "victim": "Tony Wang (Author)",
                 "description": ["The first-author of this paper (a Go novice) also managed to beat the adversary that was trained against our hardened victim, which had a pass-alive defense. The author was still able to win, demonstrating that this alternate version of our adversary is still no stronger than a Go amateur."]
+            },
+            {
+                "title": "Human analysis of hardened victim vs. adversary",
+                "dir_name": "analysis",
+                "server": "dqn.ist.berkeley.edu",
+                "paths": [
+                    "/nas/ucb/josephmiller/sgf-viewer-games/kellin-analysis-game.sgfs"
+                ],
+                "max_games": 10,
+                "adversary": "498 million training steps, 600 visits",
+                "victim": "Latest, 1600 visits, pass-alive defense",
+                "description": [
+                    "An expert-level (6 dan) human player on our team analyzed the following game. It shows typical behavior and outcomes with an adversary trained on and playing a hardened KataGo victim: the victim gains an early and soon seemingly insurmountable lead. The adversary sets a trap that would be easy for a human to see and avoid. But the victim is oblivious and collapses.",
+                    "The adversary plays non-standard, subpar moves right from the beginning. The victim's estimate of its winrate is over 90% by <a onclick='setMove(`analysis`, 9)'>move 9</a>, and a human in a high-level match would likewise hold a large advantage from this position.",
+                    "On <a onclick='setMove(`analysis`, 20)'>move 20</a>, the adversary initiates a tactic we see consistently, to produce a 'dead' (at least, according to normal judgment) square 4 group in one quadrant of the board. Elsewhere, the adversary plays low, mostly second and third line moves. This is also common in its games, and leads to the victim turning the rest of the center into its sphere of influence. We suspect this helps the adversary later play moves in that area without the victim responding directly, because the victim is already strong in that area and feels confident ignoring a number of moves.",
+                    "On <a onclick='setMove(`analysis`, 74)'>move 74</a>, the adversary begins mobilizing its 'dead' stones to set up an encirclement. Over the next 100+ moves, it gradually surrounds the victim in the top left. A key pattern here is that it leads the victim into forming an isolated group that loops around and connects to itself (a group with a cycle instead of tree structure). David 'LightVector' Wu, creator of KataGo, suggested Go-playing agents like the victim struggle to accurately judge the status of such groups, but they are normally very rare. This adversary seems to produce them consistently.",
```

Comment:
Nit: Maybe "David Wu (lightvector)" instead of "David 'LightVector' Wu"? Like, I feel like if I were referring to Joseph by his GitHub handle, it be more natural to write "Joseph Miller (UFO-101)" rather than "Joseph 'UFO-101' Miller"

Reply:
That makes sense, on second thought though I think it will be even better to remove it entirely. Just saying "David Wu, creator of KataGo" is already clear who we're giving credit to and may seem a bit more professional.

I've removed it now, let me know if you disagree though.